<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-FST0M8E" name="com.org.TripAdvisor.TripAdvisor_Automation.FlightOrder" tests="1" failures="0" timestamp="07 Feb 2018 01:22:57 GMT" time="6.011" errors="1">
  <testcase name="TestCase2" time="6.011" classname="com.org.TripAdvisor.TripAdvisor_Automation.FlightOrder">
    <error type="org.sikuli.script.FindFailed" message="C:/Users/neelk/workspace/TripAdvisor_Automation/Screenshots/Flights_Btn.JPG: (73x29) in S(0)[0,0 1920x1080] E:Y, T:3.0">
      <![CDATA[FindFailed: C:/Users/neelk/workspace/TripAdvisor_Automation/Screenshots/Flights_Btn.JPG: (73x29) in S(0)[0,0 1920x1080] E:Y, T:3.0
  Line 2759, in file Region.java

	at org.sikuli.script.Region.wait(Region.java:2759)
	at org.sikuli.script.Region.find(Region.java:2336)
	at org.sikuli.script.Region.getLocationFromTarget(Region.java:3213)
	at org.sikuli.script.Region.click(Region.java:3698)
	at org.sikuli.script.Region.click(Region.java:3683)
	at com.org.TripAdvisor.TripAdvisor_Automation.FlightOrder.TestCase2(FlightOrder.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:669)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:877)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1201)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:776)
	at org.testng.TestRunner.run(TestRunner.java:634)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- TestCase2 -->
</testsuite> <!-- com.org.TripAdvisor.TripAdvisor_Automation.FlightOrder -->
